modules/concom/sitesupport/division-parser.js

```diff
@@ -65,17 +70,23 @@ export const extractDivisionHierarchy = (departmentData) => {
       return extractDepartment(item);
     })
     .forEach((item) => {
-      const parentDivision = divisionHierarchy[item.parent_name];
+      const parentDivision = divisionHierarchy[item.parentId];
       if (parentDivision !== undefined) {
-        parentDivision.departments.push(item);
+        parentDivision.departments = [
+          ...parentDivision.departments,
+          item
+        ];
       }
     });
 
-  return Object.keys(divisionHierarchy).map((item) => {
+  // Sort Divisions and Departments alphabetically
+  const sortedDivisions = Object.keys(divisionHierarchy).map((item) => {
     return divisionHierarchy[item]
+  }).sort((a, b) => a.name.localeCompare(b.name));
+
+  sortedDivisions.forEach((division) => {
+    division.departments.sort((a, b) => a.name.localeCompare(b.name));
   });
-}
 
-// module.exports = {
-//   extractDivisionHierarchy
-// }
\ No newline at end of file
+  return sortedDivisions;
+}
```

Comment:
Again revisions to **Add data parser and test** could it get split out and integrated there?